### PR TITLE
[AssemblyScript] Second 1-bit solution

### PR DESCRIPTION
**Pull request by MaxGraey**
## Description
Second version of AssemblyScript. Now uses 1-bit Array

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeAssemblyScript/solution_2/src/sieve.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            `Passes: ${passes}\
+            , Time: ${duration}\
+            , Avg: ${duration / (passes as f32)}\
+            , Limit: ${this.size}\
+            , Count1: ${count}\
+            , Count2: ${this.countPrimes()}\
+            , Valid: ${this.validateResults()}\n`
+            .replaceAll("            ", "")
+        );
+        console.log(`maxgraey;${passes};${duration};1;algorithm=base,faithful=yes`);
```
Comment (rbergen) [inline PrimeAssemblyScript/solution_2/src/sieve.ts]: If this is indeed a solution that uses 1 bit per prime candidate, then the output line should include a `bits=1` tag, next to `algorithm=base` and `faithful=yes`.

Comment (MaxGraey) [reply PrimeAssemblyScript/solution_2/src/sieve.ts]: Thanks! Done

**Comment by rbergen** [comment]
@MaxGraey Has this PR stabilized?

**Comment by MaxGraey** [comment]
@rbergen yes. Ready for merge

**Comment by rbergen** [comment]
@MaxGraey Probably almost, but it seems not yet :)

When I run a docker build locally, I get the following output:
```
------
 > [6/6] RUN npm run build:bench:
#10 0.698
#10 0.698 > primeassemblyscript@1.0.0 build:bench
#10 0.698 > asc -O3 --noAssert --converge --initialMemory 256
#10 0.698
#10 0.705 sh: 1: asc: not found
------
```

**Comment by MaxGraey** [comment]
I apologise. Probably it's `ENV NODE_ENV production` which I add recently. Could you try again?

**Review by rbergen — APPROVED** [review]
LGTM

**Comment by MaxGraey** [comment]
Found out that it is possible to engage another experimental flag. This last change

**Review by marghidanu — APPROVED** [review]

**Pull request merged**